demo/list/list-view.html

```diff
@@ -81,34 +87,42 @@
       },
 
       animationConfig: {
-        type: Object,
-        value: function() {
-          return {
-            'entry': [{
-              name: 'fade-in-animation',
-              node: this.$.button
-            }],
-
-            'exit': [{
-              name: 'fade-out-animation',
-              node: this.$.button
-            }, {
-              name: 'hero-animation',
-              id: 'hero',
-              fromPage: this
-            }]
-          };
-        }
+        type: Object
       }
+    },
 
+    attached: function() {
+      this.animationConfig = {
+        'entry': [{
+          name: 'fade-in-animation',
+          node: this.$.button
+        }],
+
+        'exit': [{
+          name: 'fade-out-animation',
+          node: this.$.button
+        }, {
+          name: 'hero-animation',
+          id: 'hero',
+          fromPage: this
+        }]
+      };
     },
 
     _onClick: function(event) {
-      var target = event.target;
-      while (target !== this && !target._templateInstance) {
-        target = target.parentNode;
+      var eventPath = Polymer.dom(event.path);
```

Comment:
It seems that in Polymer 2.x `event.target` is not returning the localTarget (aka the items).
Can you link the bug here?
Also, there are couple typos here e.g. `Polymer.dom(event.path)` => `Polymer.dom(event).path`. The root target will be `eventPath[0]`